### DISCUSSION TITLE
Discover all repositories with given topic

### DISCUSSION THREAD

**Discussion by rrozek** [community/community#67194] [Apps, API and Webhooks] [answered]
### Select Topic Area

Bug

### Body

Hi, in my usecase i need to find all repositories marked with given topic on github. The topic in question is for instance 'astar-network'. When i head over to https://github.com/topics/astar-network i can see 16 repositories, but when i execute search query (the only way i know about to get repositories by topic) https://api.github.com/search/repositories?q=topic:astar-network i get only 15 results. the missing repo is this: https://github.com/ahmetzturk/neurolauncher and it clearly is marked as 'astar-network'.
Similar discrepancy can be visible for topic 'dappstaking': https://github.com/topics/dappstaking and https://api.github.com/search/repositories?q=topic:dAppStaking 6 to 5 and the same repo is missing.

Could it be because repository in question is actually a fork and the parent repository is not marked with given topic? How does this work?

thanks!

**Comment by shaedrich** [comment]
> There could be a few different things happening here. Without getting too knee deep in the code that decides what repositories show up on a topic page, there are several pieces of criteria that would hide them: not having any commits, being deleted, but our topics counts in different places haven't re-cached the count... I think those are two reasons that I can think of off the top of my head.
– @kenyonj via https://github.com/github/explore/pull/4018#issuecomment-1795353703

**Comment by OmBharsakle** [comment]
ombharsakel#1

**Comment by RohanVashisht1234** [comment, marked as answer]
This is because the GitHub api doesn't show forked repositories at https://api.github.com/search/repositories?q=topic:astar-network
to see all the 16 (now 40) repositories, you just need to: https://api.github.com/search/repositories?q=topic:astar-network+fork:true

If you think that my answer is correct, you can mark this as the answer.

[a comment by an automated account is not shown]
